### PR TITLE
Maint-24704 : Fix Long comments and replies

### PR DESCRIPTION
**Pull request by wafakd**
Fix long comments and replies

### CODE REVIEW

**Review by ahamdi — APPROVED** [review]
LGTM

File: platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less [hunk 1]
```diff
@@ -1791,7 +1791,8 @@ blockquote:after {
 							   padding-right: 10px;
 							 }
 							 .contentComment {
-							   display: inline;
+								word-break: break-word;
+								display: inline;
 							 }
```
Comment (ahamdi) [inline platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less:1806]:
```suggestion
							  word-break: break-word;
							  display: inline;
							 }
```

**Comment by boubaker** [comment]
Sorry, it's already considered by Meeds FT ;)
https://github.com/Meeds-io/platform-ui/pull/68
@wafakd , do you confirm that the attached PR fixes the bug ?

**Comment by ahamdi** [comment]
@boubaker I understood that this is about long words in comments , it was reported in the attached Tribe feedback issue (long link URL). Does #68 fix the same issue ? cc @wafakd @SaraBoutej

**Comment by wafakd** [comment]
> @boubaker I understood that this is about long words in comments , it was reported in the attached Tribe feedback issue (long link URL). Does #68 fix the same issue ? cc @wafakd @SaraBoutej

I think it doesn't fix the style of long URL in comment box, but we can wait for @SaraBoutej  's confirmation

**Comment by SaraBoutej** [comment]
This bug is already fixed and merged 
See this PR [https://github.com/Meeds-io/platform-ui/pull/68](https://github.com/Meeds-io/platform-ui/pull/68)

**Pull request closed without merging**